apps/cookbook/src/app/examples/slides-example/slides-code-snippets.ts

```diff
@@ -61,7 +45,7 @@ export const advancedExampleComponentHTML = `<kirby-slides
   (slideChange)="getDataFromActiveSlide($event)"
   [slidesOptions]="customConfig"
   [showNavigation]="true"
-  [title]="'Title'"
+  [title]="title"
```

Comment:
Is the `advancedExampleComponentHTML` needed at all anymore? It looks like it has been moved to `/examples/slides-advanced.component.ts `